fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterContext.java

```diff
@@ -62,8 +61,10 @@ public class RuntimeFilterContext {
 
     private final Map<PhysicalHashJoin, List<RuntimeFilter>> runtimeFilterOnHashJoinNode = Maps.newHashMap();
 
-    // Alias's child to itself.
-    private final Map<Slot, NamedExpression> aliasChildToSelf = Maps.newHashMap();
+    // alias -> alias's child, if there's a key that is alias's child, the key-value will change by this way
+    // Alias(A) = B, now B -> A in map, and encounter Alias(B) -> C, the kv will be C -> A.
+    // you can see disjoint set data structure to learn the processing detailed.
+    private final Map<NamedExpression, Pair<RelationId, NamedExpression>> aliasTransferMap = Maps.newHashMap();
```

Comment:
according to the comments, maybe Map<Pair<RelationId, NamedExpression>, NamedExpression> is better

Reply:
no, because we change key at visitPhysicalProject, use  pair as key may be more complex.